packages/app/templates/ui-extensions/projects/web_pixel_extension/README.md

```diff
@@ -0,0 +1,11 @@
+# Web Pixel Extension
+
+Web pixel app extensions provide developers with a simplified process for managing and processing behavioral data, by loading pixels in a secure sandbox environment with APIs for subscribing to customer events. Web pixel app extensions provide the following benefits to app users and developers:
+
+- Eliminate or minimize the need for users to add tracking code
+- Securely access all surfaces, like storefront, checkout and post-purchase pages
+- Control what data the developers have access to
+- Avoid performance and privacy alerts
+- Provide a smaller pixel code libraries with the removal of excess DOM manipulation code
```

Comment:
also this seems grammatically wrong but ack that you just copy-pasted from https://shopify.dev/docs/apps/marketing/pixels

Reply:
✅  and also fixed my own sentence just below it.